src/malay/AdjectiveMay.gf

```diff
@@ -6,7 +6,7 @@ concrete AdjectiveMay of Adjective = CatMay ** open ResMay, Prelude in {
 
   -- : A  -> AP ;
   PositA a = a ** {
-    compar = [] ;
+    compar = a.s ;
```

Comment:
The `a.s` is kept in the AP, because of the record extension on line 8. The field `compar` should be empty for the result of PositA. 

In contrast, you can see in ComparA that the NP part of the comparison is added into the `compar` field. (Note: I suspect that my implementation of ComparA is wrong, since it doesn't add any other word, it just becomes "warm cat" instead of "warmer than cat". So ComparA and UseComparA should be fixed too eventually.)